### ISSUE TITLE
Links are outdated for XCSoar-testing (page not found) and nightly-build (404) version

### ISSUE THREAD

**Issue by MikeJaras** [XCSoar/website#61]
This section on [link](https://www.xcsoar.org/develop/):
[Download XCSoar-testing (for Android)](https://play.google.com/store/apps/details?id=org.xcsoar.testing) or [the nightly-build version](https://download.xcsoar.org/nightly_builds/) of XCSoar, test it on the ground only, and [report bugs](https://www.xcsoar.org/develop/new_ticket.html). This version is unstable, so do not use it in flight.

**Comment by lordfolken** [comment]
Why don't you create a PR?

**Comment by lordfolken** [comment]
i don't think we have a testing version anylonger. There is still some trouble with the "nighly builds" upload, however it can be downloaded from the current build actions on github.

**Comment by MikeJaras** [comment]
> Why don't you create a PR?

Not sure what to do, give a hint and I'll give it a try.
